src/client/components/Pokedex.jsx

```diff
@@ -30,6 +30,10 @@ const Pokedex = () => {
         return pad.substring(0, pad.length - str.length) + str;
     }
     
+    const averageRating = pokemon => {
+        const ratingsTotal = pokemon.strength + pokemon.companion + pokemon.design
+        return Math.round(ratingsTotal/3)
+    }
```

Comment:
Clean, nice :)
Does the 3 count as a magic number?

Reply:
hmm, well `ratingsTotal` isnt dynamic since it has to target specific values of the rating model... 

I guess it technically is, although I dont see a clean way to fix it without going overboard on unreadable code. This is very concise and you can see what the 3 is doing, and the only time the 3 would change is if the whole function is changed. What does Steve/Nathan think?

Reply:
I suppose you could have a `ratingsArr = ["strength", "companion", "design"]` and then `ratingsTotal` reduces to sum the values, and then divide by `ratingsArr.length`

Reply:
![image](https://user-images.githubusercontent.com/90600596/155977874-b4fc714f-a015-4f79-9c31-4adf1f2d1934.png)

Reply:
You can try something like that?

Reply:
Ha! So what's the outcome? 

```
const ratingsTotal = pokemon.strength + pokemon.companion + pokemon.design
        return Math.round(ratingsTotal/3)
 ```
 For the record, this isn't really a magic number because it directly relates to the line above - i.e. you have three things contributing to the total, so you divide by three to get the average.